### ISSUE TITLE
cannot update the subscription

### ISSUE THREAD

**Issue by lcdservices** [iATSPayments/com.iatspayments.civicrm#138]
One of my clients is a big user of the ability to manage the subscription details. In particular, they generally try to schedule all recurring donations for the 1st and 16th of the month.

While they are able to make those updates for recurring payments they created through the backend, they cannot edit ones created by users on the frontend. They get the message: "You cannot update the subscription."

Is there a reason why those records cannot be modified? Is there any workaround?

**Comment by KarinG** [comment]
Are you absolutely positive the frontend generated recurring contributions are transacted with iATS as well? Reason I ask is because your client is getting the default error notification for payment processors that don't allow updates managed via CiviCRM. PS what version of CiviCRM is your client on?

**Comment by lcdservices** [comment]
Yes -- it's definitely using iATS. That's the only payment processor we're using.

I did some more testing, and occasionally there is an online recurring contrib that is editable, but the majority of those created through a contrib page are not editable. There's no clear pattern.

They are running WordPress + CiviCRM 4.6.16

**Comment by KarinG** [comment]
Credit Card and/or ACHEFT?

**Comment by lcdservices** [comment]
Credit Card.

**Comment by adixon** [comment]
There's a core bug that prevents you editing a series if there are no contributions for it. So if this case only applies to new recurring contributions, and you're deleting the initial pending one somehow, then it makes sense. You have to wait until the first real contribution goes through before deleting that pending one. In the near future, we should be matching up the first real contribution with that pending one and then you won't have to delete it at all.

**Comment by lcdservices** [comment]
You might be on to something.
It looks like most (all?) of the problem records are those with just a single contrib record (associated with the recurring profile). however -- they're not presented as pending, but completed, and the transaction did in fact get processed. see the screenshot below of a record which errors on edit.

![image](https://cloud.githubusercontent.com/assets/366478/15760942/a52827a8-28e3-11e6-8ec5-95b8b5c8acc2.png)

**Comment by adixon** [comment]
Any more clues on this issue?
